### ISSUE TITLE
Not connecting issues (errors) with performance traces

### ISSUE THREAD

**Issue by askold123** [getsentry/sentry-symfony#462]
### Environment

Using https://github.com/getsentry/sentry-symfony with Symfony 5.2

### Steps to Reproduce

Enable tracing as in sentry doc https://docs.sentry.io/platforms/php/guides/symfony/performance/#custom-instrumentation using this options in config:
```yaml
sentry:
  dsn: "%env(SENTRY_DSN)%"
  options:
    traces_sample_rate: 1.0
    sample_rate: 1
```

### Expected Result

Exceptions are connected with transaction as it shown in this video https://youtu.be/sA1rED-zb6w?t=375

### Actual Result

Exceptions are sent in separate event and are not connected to transaction in Sentry UI.

**Comment by stayallive** [comment]
So I found the issue, or at least what could cause it, @Jean85 and @ste93cry will hopefully help me out with tracking it down 💪 

Events in Sentry have a `transaction` key and for a performance tracing transactions to be matched to issues like exceptions they both need to have the same value in the `transaction` key.

I'm not sure if Symfony is setting that `transaction` key at all on the `Event` (for exceptions or the tracing span) but that is what is needed to link them together.

I have moved this issue back to the Symfony repo since this is all working in the base SDK. It might need some changes on the Symfony side to properly have the `transaction` key populated.
